remo/featuredrep/views.py

```diff
@@ -10,11 +11,25 @@
 from models import FeaturedRep
 
 
-@cache_control(private=True, max_age=60*10)
+@cache_control(private=True, max_age=60 * 10)
 def list_featured(request):
     """List all Featured Reps."""
+    featured_list = FeaturedRep.objects.all().order_by('-created_on')
```

Comment:
Yes, that's the problem. `updated_on` is not very reliable if we want to display Featured Reps by the order they got to be featured. A featured rep record could be updated at any time (eg. to fix a typo on the bio). That's why the list [currently](http://reps.mozilla.org/featured) seems almost random. `created_on` is probably closer to want we want to achieve.